templates/download/iot/index.html

```diff
@@ -115,6 +115,23 @@ <h3>Supported platforms</h3>
         </p>
       </div>
     </div>
+    <div class="row">
+      <div class="col-8">
+        <p>Ubuntu works along with silicon vendors and manufacturers to enable and certify Ubuntu on a wide range of hardware. The certified hardware passes our extensive testing and review process in our labs, ensuring that Ubuntu runs well out of the box, and regression tests are performed before every system update is delivered.</p>
+        <p><a href="https://ubuntu.com/certified" class="p-button">Browse the full list of certified hardware</a></p>
```

Comment:
David made a change on the copy doc and seems to want this button above the paragraph which is not our usual convention. Maybe that's a design or ux issue? He also changed the text to "The ultimate Ubuntu Experience: Get Certified Hardware"